### PR TITLE
#164352842 Setup react project

### PR DESCRIPTION
**Pull request by armstrongsouljah**
#### What does this PR do?
- Adds an implementation to set up a react project from scratch using webpack.

#### Description of the task to be implemented.
- Create the project root folder.
- Install project dependencies.
- Configure webpack to run transpiling and bundling.
- Setup start script.

#### How can this be manually tested?
- Clone this branch to your local machine by running `git  clone https://github.com/armstrongsouljah/storemanager-frontend.git`
- Fetch this branch to your local machine by running `git fetch origin ch-setup-react-project-164352842`
- Checkout to this branch by running `git checkout ch-setup-react-project-164352842`
- Install project dependencies by running `yarn or npm install`.
- Start the development server by running `yarn start or npm start`.
- On success, a message `welcome to storemanager` should display.

#### What are the related pivotal stories?
[164352842](https://www.pivotaltracker.com/story/show/164352842)

### CODE REVIEW

**Review by MuhweziDeo — APPROVED** [review]
LGTM

File: src/views/App.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const App = () => (
+    <div>
+        <h1>Welcome to store manager</h1>
+    </div>
+);
+
+export default App;
```
Comment (kdaglas) [inline src/views/App.jsx]: new line required

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Store Manager</title>
+</head>
+<body>
+    <div id="root"></div>
+    <script src="bundle.js"></script>
+</body>
+</html>
```
Comment (kdaglas) [inline src/index.html]: new line required

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+node_modules
+.env/
+*.lock
```
Comment (kdaglas) [inline .gitignore]: newline required

File: src/views/App.jsx (hunk 1 above)
Comment (armstrongsouljah) [reply src/views/App.jsx]: Kale hound, let me add that.

**Pull request merged**